users/urls.py

```diff
@@ -0,0 +1,7 @@
+from django.urls import path
+
+from .views import KakaoSignInView
+
+urlpatterns = [
+    path('/signin', KakaoSignInView.as_view()),
```

Comment:
지금은 일반 로그인이 없지만, 나중에 일반 로그인과 구별 될 수 있게 다음과 같이 수정하면 좋을 것 같아요.
```suggestion
    path('/signin/kakao', KakaoSignInView.as_view()),
```

Reply:
넵